index.html

```diff
@@ -5164,6 +5164,116 @@ <h2 id="AudioWorklet-concepts">
           </dd>
         </dl>
         <section>
+          <h2 id="AudioWorkletGlobalScope">
+            The <dfn>AudioWorkletGlobalScope</dfn> interface
+          </h2>
+          <p>
+            This special execution context is designed to enable the
+            generation, processing, and analysis of audio data directly using
+            JavaScript in the audio <a>rendering thread</a>. The user-supplied
+            script code is evaluated in this scope to define an
+            <a>AudioWorkletProcessor</a>.
+          </p>
+          <p>
+            <a>AudioWorkletGlobalScope</a> has a <dfn>node name to processor
+            definition map</dfn>. Initially it is empty and populated when
+            <code>registerProcessor</code> method is called.
+          </p>
+          <dl title="callback VoidFunction = void" class="idl"></dl>
+          <dl title=
+          "[Global=(Worklet, AudioWorklet), Exposed=AudioWorklet] interface AudioWorkletGlobalScope : WorkletGlobalScope"
+          class="idl">
+            <dt>
+              void registerProcessor ()
+            </dt>
+            <dd>
+              <p>
+                Registers a class definition derived from
+                <a>AudioWorkletProcessor</a>.
+              </p>
+              <p>
+                When the <a><code>registerProcessor(<em>name</em>,
+                <em>processorCtor</em>)</code></a> method is called, the user
+                agent must run the following steps:
+              </p>
+              <ol>
+                <li>If the <code><em>name</em></code> exists as a key in the
+                <a>node name to processor definition map</a>, throw a
+                <code>NotSupportedError</code> and abort all these steps.
```

Comment:
I agree with the conclusion. In my mind the analogy is to setting a new value for an existing key in any arbitrary map -- why would this be disallowed for any other map?  It might be that there is some intention to override an existing definition and although this might be awkward today, I don't think we can assume that there isn't some purpose to registering the same name twice.